internal/controllers/requirements/requirements.go

```diff
@@ -38,16 +38,20 @@ func NewRequirements() *Requirements {
 	}
 }
 
-func (r *Requirements) ValidateDocker() {
+func (r *Requirements) ValidateDocker() error {
 	err := r.dockerRequirements.ValidateDocker()
 	if err != nil {
-		logger.LogPanicWithLevel(messages.MsgPanicDockerRequirementsToRunHorusec, ErrRequirements)
+		logger.LogErrorWithLevel(messages.MsgPanicDockerRequirementsToRunHorusec, ErrRequirements)
+		return err
```

Comment:
I think that log the error and return the error would be redundant, since the log and the error message will be printed.

Reply:
the error will only by printed by cobra stack trace and not necessarily formatted as in the log, i think it would be clearer for the reader to have both logs.

Reply:
I disagree, the log error is confusing, there is a lot of information. Just printing the error is more clear and easily to read, but ok.